packages/fxa-auth-server/package.json

```diff
@@ -66,7 +66,6 @@
     "jed": "0.5.4",
     "joi": "14.0.0",
     "jsonwebtoken": "^8.4.0",
-    "keypair": "1.0.1",
```

Comment:
Node has `crypto.generateKeyPair` natively these days, we don't need to pull in an extra dep for it.